internal/infrastructure/kubernetes/proxy_deployment.go

```diff
@@ -133,9 +133,19 @@ func expectedProxyContainers(infra *ir.Infra) ([]corev1.Container, error) {
 		},
 	}
 
-	cfg, err := bootstrap.GetRenderedBootstrapConfig()
-	if err != nil {
-		return nil, err
+	var cfg string
+	// Get Bootstrap from EnvoyProxy API if set by the user
+	// The config should have been validated already
+	if infra.Proxy.Config != nil &&
+		infra.Proxy.Config.Spec.Bootstrap != nil {
+		cfg = *infra.Proxy.Config.Spec.Bootstrap
```

Comment:
I think we should unmarshal it to bootstrap struct to validate it if the bootstrap user provided is invalid.

Reply:
the validation will happen in the validation webhook, and the data should be valid by the time it reaches this component